packages/strapi-plugin-upload/admin/src/containers/HomePage/index.js

```diff
@@ -183,16 +223,15 @@ const HomePage = () => {
       {
         id: getTrad(getHeaderLabel(data)),
```

Comment:
Do you have an error if data is empty?

Reply:
No I don't have an error because getHeaderLabel() also return a string if array.length is 0 :

```
const getHeaderLabel = array => {
  let headerLabel = 'header.content.assets-single';

  if (array.length === 0) {
    headerLabel = 'header.content.assets-empty';
  } else if (array.length > 1) {
    headerLabel = 'header.content.assets-multiple';
  }

  return headerLabel;
};
```